src/main/java/Duke.java

```diff
@@ -1,10 +1,68 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        System.out.println("____________________________________________________________\n" +
+                "Hello! I'm Duke\n" +
```

Comment:
```suggestion
                + "Hello! I'm Duke\n"
```
Try to include the operator in the front so that readers do not need to refer back to the previous line